pyriemann/classification.py

```diff
@@ -125,7 +109,9 @@ def fit(self, X, y, sample_weight=None):
         self : MDM instance
             The MDM instance.
         """
-        self.metric_mean, self.metric_dist = _check_metric(self.metric)
+        self.metric_mean, self.metric_dist = check_metric(
+            self.metric, ["mean", "distance"]
```

Comment:
I think you can just use:
```
check_metric(self.metric)
```
as `["mean", "distance"]` is already your default.
(same below)

Reply:
Yes, good remark!